src/ButtonListener.java

```diff
@@ -1,16 +0,0 @@
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
-import javax.swing.JTextArea;
-
-public class ButtonListener implements ActionListener {
-
-	public ButtonListener(JTextArea origin, JTextArea destination) {
-
-	}
-
-	@Override
-	public void actionPerformed(ActionEvent ae) {
-		Main.PlayAudio();
-
-	}
-}
```

Comment:
I deleted this class. It is unnecessary. I moved all methods to MessageListener class and changed them a little bit.